src/frontends/tensorflow_common/src/op/binary_op.cpp

```diff
@@ -171,6 +171,33 @@ OutputVector translate_addv2_op(const NodeContext& node) {
     return {result};
 }
 
+OutputVector translate_sub_op(const NodeContext& node) {
+    default_op_checks(node, 2, {"Sub"}, true);
+    auto lhs = node.get_input(0);
+    auto rhs = node.get_input(1);
+
+    auto complex_type_mark_lhs = as_type_ptr<ComplexTypeMark>(lhs.get_node_shared_ptr());
+    auto complex_type_mark_rhs = as_type_ptr<ComplexTypeMark>(rhs.get_node_shared_ptr());
+    auto complex_type_inputs = (complex_type_mark_lhs && complex_type_mark_rhs);
+
+    if (complex_type_inputs) {
+        lhs = complex_type_mark_lhs->input_value(0);
+        rhs = complex_type_mark_rhs->input_value(0);
+    }
+
+    // performing an actual operation
+    auto result = make_shared<v1::Subtract>(lhs, rhs);
+
+    if (complex_type_inputs) {
+        auto complex_result = make_shared<ComplexTypeMark>(result, complex_type_mark_lhs->get_complex_part_type());
+        set_node_name(node.get_name(), result);
+
+        return {complex_result};
+    }
+    set_node_name(node.get_name(), result);
+    return {result};
+}
+
 template OutputVector translate_binary_op<v1::Add>(const NodeContext& node);
```

Comment:
please remove `template OutputVector translate_binary_op<v13::BitwiseAnd>(const NodeContext& node);` and `template OutputVector translate_binary_op<v1::Subtract>(const NodeContext& node);`
They look no longer needed.

Reply:
after deleting these lines i get
`
openvino/bin/intel64/Release/libopenvino_tensorflow_lite_frontend.so.2024.4.0: undefined reference to std::vector<ov::Output<ov::Node>, std::allocator<ov::Output<ov::Node> > > ov::frontend::tensorflow::op::translate_binary_op<ov::op::v1::Subtract>(ov::frontend::NodeContext const&)
`

Reply:
Also, it seems like `BitwiseAnd` in `op_table.cpp` still looks like this `{"BitwiseAnd", CreatorFunction(translate_binary_op<v13::BitwiseAnd>)},`. 

Did you meant to delete template `OutputVector translate_binary_op<v1::Add>(const NodeContext& node);` instead?

Reply:
https://github.com/openvinotoolkit/openvino/blob/04e0fea532159d0993b0af5747703db8c4825ffe/src/frontends/tensorflow_lite/src/op/op_translation_utils.hpp#L50-L62

https://github.com/openvinotoolkit/openvino/blob/04e0fea532159d0993b0af5747703db8c4825ffe/src/frontends/tensorflow/src/op_table.cpp#L198

I assume this part in tensorflow_lite might be the reason for the error. Should I update Sub there?

Reply:
Yes, you look right. So let us leave it as is